agent/logging.go

```diff
@@ -90,63 +90,93 @@ func GetSender(ctx context.Context, prefix, taskId string) (send.Sender, error)
 func (a *Agent) makeLoggerProducer(ctx context.Context, tc *taskContext, c *model.LoggerConfig, commandName string) client.LoggerProducer {
 	path := filepath.Join(a.opts.WorkingDirectory, taskLogDirectory)
 	grip.Error(errors.Wrap(os.Mkdir(path, os.ModeDir|os.ModePerm), "error making log directory"))
+	config := a.convertLoggerConfig(tc, c)
 
+	logger := a.comm.GetLoggerProducer(ctx, tc.task, &config)
+	loggerData := a.comm.GetLoggerMetadata()
+	tc.logs = &apimodels.TaskLogs{}
+	for _, agent := range loggerData.Agent {
+		tc.logs.AgentLogURLs = append(tc.logs.AgentLogURLs, apimodels.LogInfo{
+			Command: commandName,
+			URL:     fmt.Sprintf("%s/build/%s/test/%s", a.opts.LogkeeperURL, agent.Build, agent.Test),
```

Comment:
is it always logkeeper as the base URL?

Reply:
It's a configuration option but for us it'll be logkeeper until we have v3